draft-ietf-quic-transport.md

```diff
@@ -2302,6 +2302,12 @@ the application requests that the connection be closed.  The application
 protocol can use an APPLICATION_CLOSE message with an appropriate error code to
 signal closure.
 
+If the endpoint believes that a connection has been successfully established, it
```

Comment:
I find this condition a little vague for a MUST. How about "has received a short header from the peer"?